InvenTree/order/migrations/0099_alter_salesorder_status.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.11 on 2024-03-20 20:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('order', '0098_auto_20231024_1844'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='salesorder',
+            name='status',
+            field=models.PositiveIntegerField(choices=[(10, 'Pending'), (15, 'In Progress'), (20, 'Shipped'), (30, 'Complete'), (40, 'Cancelled'), (50, 'Lost'), (60, 'Returned')], default=10, help_text='Purchase order status', verbose_name='Status'),
```

Comment:
This should be adapted to work like below-linked line to ensure we do not need migrations on states in the future.

https://github.com/matmair/InvenTree/blob/eecda20b83e42b10b43b12e78ebb4c38f26916c7/InvenTree/stock/migrations/0102_alter_stockitem_status.py#L18

Reply:
Agreed - please make the change as suggested here @tsimonq2

Reply:
How does my recently-pushed commit look for this? @SchrodingersGat @matmair

Reply:
Looks good @tsimonq2 !